client/public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
Whats happening with this `%PUBLIC_URL%`? What does it do? Where is it set/pulled from?

Reply:
This expects to be defined by the environment, which hasn't been set here in our development environment. When deploying to a service like Heroku this may change. If we wanted to specify a value in its place while still in development, we could set a "homepage" attribute in package.json.

Is this acceptable as is?

Reply:
Ah, yup makes sense. Would using a relative URL work?
If not we could look to set a dev value to `localhost:port` if its not already. We ideally would have our dev environment mimic as closely as we can to our prod env. It reduces issues that only happen in production.